src/fpu/fpu_instructions.h

```diff
@@ -102,9 +103,33 @@ static void FPU_FPOP(void){
 static double FROUND(double in){
 	switch(fpu.round){
 	case ROUND_Nearest: return std::nearbyint(in);
-	case ROUND_Down: return (floor(in));
-	case ROUND_Up: return (ceil(in));
-	case ROUND_Chop: [[fallthrough]]; // cast by the caller chops
+	case ROUND_Down: return std::floor(in);
+	case ROUND_Up: return std::ceil(in);
+	case ROUND_Chop: 
+	{
+		// Most x32/x64 builds use fpu_instructions_x86.h.
+		// If we are here on x64, we don't need this fix.
+#if !defined(__x86_64__) && !defined(_M_X64)
```

Comment:
@kcgen let me know if you are ok with this here, or if you would prefer a new build system define get created and used instead.

Reply:
Good point - if the CPU type isn't commonly defined, Meson could genericize them for us.  Seeing them used in the DR codecs, XXHash, and STB Vorbis libs.. and these are extremely universal headers compiled by countless compilers; so I think these are defacto enough.

Reply:
No, this can't be correct. As I explained below, the bug 100% manifest in our 64-bit x86 builds—I tried it yesterday on Win10. I also tried the 0.80.1 32-bit build, and the 32-bit x86 build is "fine" because that uses 80-bit FPU regs internally by fluke.